SignalServiceKit/src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -347,7 +347,7 @@ - (nullable UIImage *)thumbnailImage
     }
 
     if (![[NSFileManager defaultManager] fileExistsAtPath:thumbnailPath]) {
-        OWSFail(@"%@ missing thumbnail for attachmentId: %@", self.logTag, self.uniqueId);
+        DDLogError(@"%@ missing thumbnail for attachmentId: %@", self.logTag, self.uniqueId);
```

Comment:
When does this happen where we'd want to ignore this assertion? Is it during a restore? can you leave a comment?